tests/errors/test_dict_const.py

```diff
@@ -0,0 +1,7 @@
+from lpython import i32, f64, str, dict, Const
+
+
+def test_dict_const():
+    d: Const[dict[str, i32]] = {"a": 1, "b": 2, "c": 3}
+    print(d.pop("a"))
+    i: i32 = d.pop("a")
```

Comment:
Add this tests to tests.toml and do `./run_tests.py -u`

Reply:
Hmm..so we need to do this manually? I thought `./run_tests.py -u` does this automatically. How about adding this information to the docs?

Reply:
It was present in the README.md, I think someone has remove the tests section in README.md

Reply:
It was me only @Thirumalai-Shaktivel . @certik suggested to keep it short in the README and redirect user to the installation docs for more. The testing section is there.

Reply:
Yes, understood. Now, it looks neat.
Thanks for doing it.

Reply:
> How about adding this information to the docs?

Yes, let's add it.